lib/util/action.js

```diff
@@ -21,7 +21,7 @@ module.exports = (key, isSelect) => {
   if (key.name === 'backspace') return 'delete';
   if (key.name === 'delete') return 'deleteForward';
   if (key.name === 'abort') return 'abort';
-  if (key.name === 'escape') return 'abort';
+  if (key.name === 'escape') return 'exit';
```

Comment:
The idea was: https://stackoverflow.com/a/2820407
- abort indicates "abnormal" end to the program
- exit indicates a "normal" end to the program